templates/user/dashboard/feeds.tmpl

```diff
@@ -13,8 +13,8 @@
 						{{else if eq .GetOpType 2}}
 							{{$.i18n.Tr "action.rename_repo" .GetContent .GetRepoLink .ShortRepoPath | Str2html}}
 						{{else if eq .GetOpType 5}}
-							{{ $branchLink := .GetBranch | EscapePound}}
-							{{$.i18n.Tr "action.commit_repo" .GetRepoLink $branchLink .GetBranch .ShortRepoPath | Str2html}}
+							{{ $branchLink := (printf "branch/%s" .GetBranch) | EscapePound | Escape}}
```

Comment:
I think this should go into the translation file that already holds a part of the url.

Reply:
Is it really a good idea to put even more code there that is redundant across all translations? If so, am I supposed to edit the en-US file so that all translations have to be adapted via crowdin?

Reply:
I just don't like that one part is in the locale file and another one in code. It should be either or but not both.
Yes, just commit the en-US file with the changes and crowdin will spread the word that there are new changes that need translations